lib/Twig/Loader/Filesystem.php

```diff
@@ -19,9 +19,9 @@ class Twig_Loader_Filesystem implements Twig_LoaderInterface
     /** Identifier of the main namespace. */
     const MAIN_NAMESPACE = '__main__';
 
-    protected $paths = array();
-    protected $cache = array();
-    protected $errorCache = array();
+    private $paths = array();
+    private $cache = array();
+    private $errorCache = array();
```

Comment:
btw, I think that the Symfony loader is broken (for the bundle notation) for recent versions of Twig when using a Twig_Loader_Chain or multiple templating engines in symfony (i.e. anything using the ``exists`` method to find the right loader supporting the template) as it extends ``findTemplate`` to implement the logic but Twig_Loader_Filesystem was refactored so that findTemplate uses exists rather than the opposite